services/agent/src/simcore_service_agent/services/backup.py

```diff
@@ -107,6 +112,15 @@ def _log_expected_operation(
     _logger.log(log_level, formatted_message)
 
 
+async def _ensure_permissions_on_source_dir(source_dir: Path) -> None:
+    self_container = os.environ["HOSTNAME"]
+    await run_command_in_container(
+        self_container,
+        command=f"chmod -R o+rX '{source_dir}'",
```

Comment:
please use long option names

Reply:
tried this one, recursive is not available for all containers. will keep short version which is known to most